package.json

```diff
@@ -17,7 +17,7 @@
     "copy-assets": "mkdir -p dist && copyfiles -u 1 src/**/*.njk dist/ src/locales/**/** dist/ && copyfiles -u 3 src/assets/javascripts/*.js dist/public/scripts && copyfiles -u 3 src/assets/images/* dist/public/images",
     "build-js": "yarn build-js:application && yarn build-js:analytics && yarn build-js:all",
     "build-js:application": "mkdir -p dist/public/javascripts; uglifyjs src/assets/javascripts/application.js --beautify -o dist/public/javascripts/application.js",
-    "build-js:analytics": "mkdir -p dist/public/javascripts; uglifyjs src/assets/javascript/analytics/**/*.js src/assets/javascript/analytics/*.js node_modules/di-ipv-cri-common-express/src/assets/javascript/cookies.js --beautify -o dist/public/javascripts/analytics.js",
+    "build-js:analytics": "mkdir -p dist/public/javascripts; uglifyjs src/assets/javascript/analytics/**/*.js src/assets/javascript/analytics/*.js node_modules/@govuk-one-login/di-ipv-cri-common-express/src/assets/javascript/cookies.js --beautify -o dist/public/javascripts/analytics.js",
```

Comment:
didn't we remove the prefix di- from the repository names?

Reply:
This is the [published package in npm](https://www.npmjs.com/package/@govuk-one-login/di-ipv-cri-common-express), not the repository name. The publish package has not currently been renamed as it was published before the org migration happened.

The package rename will be a future PR across all of the CRIs.